ultravox/training/configs/llama_whisper.yaml

```diff
@@ -0,0 +1,103 @@
+# llama3.1-8b + whisper-medium, for development
+
+exp_name: "llama3.1-8b-whisper"
+text_model: "meta-llama/Meta-Llama-3.1-8B-Instruct"
+audio_model: "openai/whisper-medium"
+
+loss_config:
+  loss_function: "KL_Divergence"
+
+max_steps: 20 # x8x24 = 2,764,800
+
+# This would go in a datasets.yaml file and we could either use pyyaml-include to include it
```

Comment:
Oh nice, I was looking for a way to include YAML files, assuming it works with `simple_parsing`.